app/UseCases/Institution/GetInstitutionsUseCase.js

```diff
@@ -0,0 +1,23 @@
+const NotFoundException = use('App/Exceptions/NotFoundException');
+
+class GetInstitutionsUseCase {
+  static get inject() {
+    return ['App/Models/Institution'];
+  }
+
+  constructor(institutionModel) {
+    this.institutionModel = institutionModel;
+  }
+
+  async execute() {
+    const institutions = await this.institutionModel.all();
+
+    if (!institutions) {
```

Comment:
Retirar validação pois quando não for encontrado nenhuma instituição sera retornado um array vazio

Reply:
ok